### PR TITLE
Contextpush

### PR DESCRIPTION
**Pull request by marcosrdz**

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/jq8bhmzjp8eqtygpwnzd69wear

[download apk](http://filestorage.bluewallet.io:1488/1603463803.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/yer3ub1upjnvtkx37732ckyw3w

[download apk](http://filestorage.bluewallet.io:1488/1603471372.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/y54qf1e86j02zgbwzkq2cv78ym

[download apk](http://filestorage.bluewallet.io:1488/1603506468.apk)

**Review by Overtorment — CHANGES_REQUESTED** [review]
can you pls fix conflicts?

**Comment by Overtorment** [comment]
linting & deepscan

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/2ud47h6jquh56k6b5brja4xdfg

[download apk](http://filestorage.bluewallet.io:1488/1603736059.apk)

File: screen/wallets/transactions.js [hunk 1]
```diff
@@ -339,7 +339,7 @@ const WalletTransactions = () => {
         wallet.current.getBalance() > 0 ? (
           <TouchableOpacity
             onPress={async () => {
-              Linking.openURL('https://bluewallet.io/marketplace/');
+              Linking.openURL('https://bluewallet.current.io/marketplace/');
```
Comment (Overtorment) [inline screen/wallets/transactions.js:342]: oops

File: blue_modules/notifications.js [hunk 2]
```diff
@@ -46,22 +46,14 @@ function Notifications(props) {
 
         // (required) Called when a remote is received or opened, or local notification is opened
         onNotification: async function (notification) {
+
```
Comment (Overtorment) [inline blue_modules/notifications.js:49]: `'Platform' is defined but never used`

**Review by Overtorment — CHANGES_REQUESTED** [review]
minor changes. 

cant validate the logic  change till I test it myself, so lets just merge and test on real devices

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/dvr0ztc8gngkv513rh28rv5fxw

[download apk](http://filestorage.bluewallet.io:1488/1603996603.apk)

File: package.json [hunk 3]
```diff
@@ -139,7 +139,7 @@
     "react-native-popup-menu-android": "1.0.3",
     "react-native-privacy-snapshot": "git+https://github.com/BlueWallet/react-native-privacy-snapshot.git",
     "react-native-prompt-android": "git+https://github.com/BlueWallet/react-native-prompt-android.git#2073b07f64bf5dc95807f64d79f37bdb111e6951",
-    "react-native-push-notification": "5.1.1",
+    "react-native-push-notification": "6.1.2",
```
Comment (Overtorment) [inline package.json:142]: i remember i was reading changelog for this package a while ago, and it said something that after update I would have to mess with configuring notification channel on android, which I kinda dont wana do unless really have to

File: blue_modules/notifications.js [hunk 4]
```diff
@@ -76,9 +67,9 @@ function Notifications(props) {
             notifications.push(payload);
             await AsyncStorage.setItem(NOTIFICATIONS_STORAGE, JSON.stringify(notifications));
           } catch (_) {}
-
+          props.onProcessNotifications();
           // (required) Called when a remote is received or opened, or local notification is opened
-          notification.finish(PushNotificationIOS.FetchResult.NoData);
+          notification.finish(PushNotificationIOS.FetchResult.NewData);
```
Comment (Overtorment) [inline blue_modules/notifications.js:72]: alright, whats this then?

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/7g0w395eebrqfdvv8vcb3jngxr

[download apk](http://filestorage.bluewallet.io:1488/1604530189.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/33fkk0cm6q4q40fnza3qqy38dg

[download apk](http://filestorage.bluewallet.io:1488/1604533734.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/btevyn4tknvajdque3jdbve2vm

[download apk](http://filestorage.bluewallet.io:1488/1604541667.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/qt4e2qk5a25zneym6z1x63f19g

[download apk](http://filestorage.bluewallet.io:1488/1604633888.apk)

**Comment by Overtorment** [comment]
@marcosrdz are you going to work on this? Im eager to close it

**Comment by marcosrdz** [comment]
> @marcosrdz are you going to work on this? Im eager to close it

Close it. Doesnt work well with ios

**Pull request closed without merging**